src/components/UserTable.js

```diff
@@ -32,6 +51,12 @@ class UserTable extends React.Component {
                         {this.renderUser(this.props.data)}
                     </tbody>
                 </table>
+                <h3 className="font-weight-bold">User Details</h3>
```

Comment:
I would prefer it if you bring this component up. `UserInfo` should be rendered from `Users.js`, not from `UserTable.js`. So `UserTable` only need to display the table itself, unlike right now is rendering the table *and* the info. Also lift up the state & the functions that it need.